Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -427,6 +427,24 @@ void ImageBuffer::waitForAcceleratedRenderingFenceCompletion()
     if (auto* backend = ensureBackend())
         backend->waitForAcceleratedRenderingFenceCompletion();
 }
+
+const GrDirectContext* ImageBuffer::directContext() const
+{
+    auto* backend = ensureBackend();
+    if (!backend)
+        return nullptr;
+    return backend->directContext();
+}
+
+RefPtr<ImageBuffer> ImageBuffer::copyAcceleratedImageBufferBorrowingBackendRenderTarget() const
+{
+    ASSERT(renderingMode() == RenderingMode::Accelerated);
+
+    auto* backend = ensureBackend();
+    ASSERT(backend);
```

Comment:
So, in directContext we handle backend being null, but here we assert instead. What's the difference?